server/apps/cli/src/app.module.ts

```diff
@@ -6,11 +6,7 @@ import { DisablePasswordLoginCommand, EnablePasswordLoginCommand } from './comma
 import { PromptPasswordQuestions, ResetAdminPasswordCommand } from './commands/reset-admin-password.command';
 
 @Module({
-  imports: [
-    DomainModule.register({
-      imports: [InfraModule],
-    }),
-  ],
+  imports: [DomainModule.register({ imports: [InfraModule] })],
```

Comment:
Not entirely sure yet how the module/imports stuff works, but why can't we just add the `ConfigModule` as an import here?

Reply:
We could, but the config module is actually needed in the (1) cli, (2) server, and (3) microservices modules. It's common infrastructure for all three entrypoints, so I just imported it the `InfraModule` itself.